app/controllers/spree/reviews_controller.rb

```diff
@@ -23,11 +23,14 @@ def create
     @review.locale = I18n.locale.to_s if Spree::Reviews::Config[:track_locale]
 
     authorize! :create, @review
+
     if @review.save
-      flash[:notice] = Spree.t(:review_successfully_submitted)
-      redirect_to spree.product_path(@product)
-    else
-      render :new
+      # flash[:notice] = Spree.t(:review_successfully_submitted)
```

Comment:
Please uncomment the flash message. We need to show the flash message on Success.

Reply:
this flash was delete because it was display through js file. 
it creates issue in html request when refresh the page after submit the form.